paintRoomCalc_human.cr

```diff
@@ -39,7 +39,7 @@ def calculate_num_layers(width : Float64, depth : Float64, height : Float64, lay
   curr_width, curr_depth, curr_height = width, depth, height
   curr_room_volume = room_volume
   # Iterate layers of paint until volume is zero or negative
-  until curr_room_volume <= 0
```

Comment:
is the break condition for this loop, it iterates 1 too many times, because i assume it rounds up. By setting  the `until` break condition to check against `curr_room_volume.floor` (with floor method), forces a round down, effectively removing the assumption to round up, and making the `until` loop not over-iterate.
This makes the `return` value of `num_floor` not over shoot, and mirror the output from original type-inferred version I wrote.